src/components/BookDetail/_BookDetail.scss

```diff
@@ -0,0 +1,124 @@
+.book-details-container {
```

Comment:
I have made a couple of changes:

- Since left-right is viewport dependend, let's try to refrain from this.
- Instead of using @media for media queries, we can use the @include media-breakpoint(<size>) mixin.

This is a good start, but most of the page is not responsive yet.
This is fine for now, but should be changed duely.